app/experimenter/nimbus-ui/src/components/TableOverview/index.tsx

```diff
@@ -57,7 +57,10 @@ const TableOverview = ({ experiment }: TableOverviewProps) => {
               <h3 className="h6">Probe Sets</h3>
               {experiment.primaryProbeSets?.length
                 ? experiment.primaryProbeSets.map((probeSet) => (
-                    <span key={probeSet?.name}>{probeSet?.name}</span>
+                    <span key={probeSet?.name}>
+                      {probeSet?.name}
+                      <br />
```

Comment:
If this needs a line-break, should this be a `p` tag instead of a span + break? (or `margin-bottom` class)

Reply:
I think the `<p>` tag adds more space than looks visually appealing, line break is a little less. margin bottom did't seem to do much on spans?  maybe it works on divs...

Reply:
I changed to `div` and it seems better